lib/electric/util.ex

```diff
@@ -21,4 +21,10 @@ defmodule Electric.Util do
     [:green, :bright, message, :reset]
     |> IO.ANSI.format()
   end
+
+  def verbose(message, opts) do
+    if opts[:verbose] do
```

Comment:
Curious, is it possible to do smth like `Application.put_env` at command parsing if a verbose flag is present and use that as a global injection to avoid passing around options like this? I don't like that we're passing verbosity levels through so many functions when it feels like it should just be injected across the app (as `Logger.configure(level: :debug)` for example)

Reply:
so I committed the sin of being too functional 😊  

you're absolutely right, passing the flags around felt weird and was a total pain